mobile-app/app/screens/AppNavigator/screens/Balances/BalancesScreen.tsx

```diff
@@ -150,3 +193,126 @@ function BalanceItemRow ({
     </ThemedTouchableOpacity>
   )
 }
+
+interface DFIBalanceCardProps {
+  utxo: WalletToken
+  token: WalletToken
+  onConvertPress: () => void
+  onSendPress: () => void
+  onUtxoVsTokenPress: () => void
+  isLight: boolean
+}
+
+function DFIBalanceCard (props: DFIBalanceCardProps): JSX.Element {
+  const DFIIcon = getNativeIcon('_UTXO')
+  const totalDFI = new BigNumber(props.utxo.amount).plus(new BigNumber(props.token.amount)).toFixed(8)
+
+  return (
+    <ThemedView
+      light={tailwind('bg-white border-gray-100')}
+      dark={tailwind('bg-gray-800')}
+      style={tailwind('mx-2 mt-4 rounded-lg flex-1')}
+      testID='dfi_balance_card'
+    >
+      <ImageBackground
+        source={props.isLight ? DFIBackground : DFIBackgroundDark}
+        style={tailwind('flex-1 rounded-lg overflow-hidden')}
+        resizeMode='cover'
+        resizeMethod='scale'
+      >
+        <View style={tailwind('flex-col flex-1 mx-4 mt-5 mb-4')}>
+          <View style={tailwind('flex-row pb-3 items-center')}>
+            <DFIIcon width={24} height={24} style={tailwind('mr-2')} />
+            <ThemedText style={tailwind('pr-9 text-lg font-bold')} testID='total_dfi_label'>DFI</ThemedText>
+            <ThemedText testID='total_dfi_amount'>{totalDFI} DFI</ThemedText>
```

Comment:
totalDFI should be NumberText right?

Reply:
We have a NumberText component?

Reply:
NumberFormat I think, from react-number-format. It adds proper formatting to numbers (e.g, comma separated etc.)